src/main/java/io/debezium/connector/spanner/kafka/internal/TaskSyncPublisher.java

```diff
@@ -34,20 +35,24 @@ public class TaskSyncPublisher {
     private volatile Instant lastTime;
     private final BufferedPublisher<TaskSyncEvent> bufferedPublisher;
     private final Consumer<RuntimeException> errorHandler;
+    private final TaskSyncContextHolder taskSyncContextHolder;
```

Comment:
If this is only passed to the BufferedPublisher, is it necessary to store a class reference here?

Reply:
Removed, thanks!